src/common/components/layouts/pages/status/JobStatus.js

```diff
@@ -70,6 +70,20 @@ function JobStatus({
     cancellingJob,
 }) {
     const { id: jobId } = useParams();
+    const [isEscapeBeforeCreate, setIsEscapeBeforeCreate] = useState(false);
+
+    useEffect(() => {
+        const cancelBeforeCreate = () => setIsEscapeBeforeCreate(true);
+        const cancelAfterCreate = () => onCancel();
+        if (jobStatus === JOB_STATUS.CREATED) document.addEventListener('visibilitychange', cancelBeforeCreate);
```

Comment:
We should cancel job only of tab was closed. If we use `visibilitychange` event, we cancel job even if user just switched from one tab to another (without closing it) or minimized the browser window.